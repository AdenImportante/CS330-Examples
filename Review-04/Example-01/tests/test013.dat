#! /bin/sh

# Test entire program with only Consumable input

cat <<EOF >test013.in
Potion Speed-II-Potion Spd*2 1
Disposable PotatoCamera Focus-50% 99
EOF

# Strip all spaces and newlines
# convert to all lowercase
./storage test013.in 4 | sed 's/[ ]//g' | grep -vE "^$" | tr '[:upper:]' '[:lower:]'