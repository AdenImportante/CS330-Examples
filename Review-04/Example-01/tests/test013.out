processinglog:
(s)speed-ii-potion
(s)potatocamera
playerstoragesummary:
-used50%of4slots
nme:speed-ii-potion
eft:spd*2
use:1
qty:1
nme:potatocamera
eft:focus-50%
use:99
qty:1
